handlers.go

```diff
@@ -4594,6 +4583,11 @@ func (s *Server) handleContentHealthCheck(c echo.Context) error {
 		return err
 	}
 
+	var exch exchange.Interface
```

Comment:
please revert this, the intention is for the walk below. You can get the exchange in the `createAggregate` func